### ISSUE TITLE
Piano Hand Hardware Design Review

### ISSUE THREAD

**Issue by Rugved-Dikay** [purdue-arc/arc-piano-hand#7]
A formal report of all hardware design progress and ideation for the Piano Hand.

**Comment by Rugved-Dikay** [comment]
To start off this semester, me and Revanth have decided to start the development of a hardware design review document/report. In general, this report will be a compilation of the design process we have used throughout the semester, any research performed into different systems and processes, and of plans for continuous development and integration of the project. As of right now the structure has yet to be finalized but will be soon, with weekly assignments given to y'all alongside whatever CAD work is needed, such as like writing one to two paragraphs about a new design choice the team made, or actuation research. Ultimately by the end of the sem, we should hopefully have a foundation of an industry standard report for ARC, and all relevant stakeholders, including ourselves, to truly cherish. Thus I wanted to make this project an ongoing issue within GitHub and will soon be making follow-up posts regarding the matter. 



Why a paper?

The purpose of this deliverable is to not only create a record of the Piano Hand's progress as a metric for ARC and planning future project work, but mainly to offer all of us a professional document to validate our hard work, and show off to recruiters, companies, and professors alike in the future. Per personal experiences, I can attest to the value of having a paper/report under your name, especially as this early stage in our undergraduate and professional lives, as a tool of distinguishing ourselves from other candidates and demonstrating to engineering and tech firms, how we have already developed a lot of the skills they are looking for: Technical writing, research, collaboration to name a few. Moreover, this will be a chance for us to communicate the research and knowledge we have gained through this project in a very professional and marketable manner, as yet another edge offered for our career development. 

 However, this document will not have the standards needed to offer the many things I have mentioned, unless we all chip in, and so I really hope you all can contribute in some way, beyond the amazing work we will be doing with CAD and actual hardware.
